components/AppBar/README.md

```diff
@@ -1,12 +1,13 @@
 <!--docs:
-title: "App Bar"
+title: "App Bars"
```

Comment:
Emily says plural for consistency with spec, although she can see an argument for singular.